kernelci.org/content/en/docs/org/board.md

```diff
@@ -17,19 +17,18 @@ year and voted by the board.
 There is also a Project Manager role to help coordinating things with the wider
 team.
 
-As of today, the Advisory Board is composed of the members listed below with
+As of 2024, the Advisory Board is composed of the members listed below with
 their respective roles, email address and IRC nicknames:
 
 * [Chris Paterson](mailto:<chris.paterson2@renesas.com>) (CIP, Treasurer) - `patersonc`
```

Comment:
Just to help clarify what CIP actually is, i.e. not an acronym relating to KernelCI

```suggestion
* [Chris Paterson](mailto:<chris.paterson2@renesas.com>) (Civil Infrastructure Platform project, Treasurer) - `patersonc`
```

Reply:
Were you going to make this change?

Reply:
Yes sorry, looks like I forgot to push the change last week.  It should all be there now.